views/partials/_sub-navigation-items.njk

```diff
@@ -0,0 +1,13 @@
+{% macro subNavigationItems(params) %}
+  {% if params.items %}
+      <ul class="app-navigation__list js-app-navigation__list--hidden">
+        {% for item in params.items %}
```

Comment:
**issue** Accordions in the current navigation have a first item linking to the section, replacing the link that's getting hidden.